shared/app.js

```diff
@@ -73,7 +74,9 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
in the `05_requirejs` example it does still work with: `var ClientRouter = this.options.ClientRouter || require('app/router');`

is there a reason we need to define it further up the chain that i'm missing?